mautic/mautic
# Comma typo

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | /
| Related developer documentation PR URL | /
| Issue(s) addressed                     | 




#### Description:

Just fixing a typo in a tooltip.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create a new form
3. Add a field of the type "File"
4. Go to the Properties tab and hover over the tooltip for Allowed file extensions
Expected: Should say "Comma separated values (e.g. 'gif,jpg')
